database.go

```diff
@@ -31,6 +32,8 @@ type BadgerDB struct {
 	lock   sync.RWMutex
 }
 
+type KVList = badger.KVList
```

Comment:
this does a type alias for this package, but other impkementations that use the interface will still need to import badger

Reply:
Wouldn't an implementation that use this import chaindb, and that would import badger?  In gossamer dot/state/storage_notify_test.go Example function, I do a Subscribe call using chaindb.KVList, and I didn't need to import badger.  Isn't that the desired result?

Reply:
hmm, I see. ok this works for now